efficientdet/aug/autoaugment.py

```diff
@@ -38,6 +37,10 @@
 except ImportError:
   from tensorflow.contrib import image as image_ops  # pylint: disable=g-import-not-at-top
 
+try:
+  import tensorflow_probability as tfp
+except ImportError:
+  import tensorflow.distributions as tfp
```

Comment:
Could you add a simple comment to these two comments? (which one is recommended by tf2?)